mLRS/Common/frames.h

```diff
@@ -428,4 +430,33 @@ tTxCmdFrameRxParams* rx_params = (tTxCmdFrameRxParams*)frame->payload;
 #endif
 
 
+//-------------------------------------------------------
+// Helper
+//-------------------------------------------------------
+
+// Numerical Recipe's quick generator randq1()
+uint32_t nr_randq1(void)
+{
+    static uint32_t seed = 0;
+    seed = 1664525UL * seed + 1013904223UL;
+    return seed;
+}
+
+
+uint8_t fhss_band_next(void)
+{
+    static uint8_t fhss_band = 0;
+    static uint8_t fhss_band_last = 0;
+
+    if (fhss_band == fhss_band_last) { // we had it two times, so toggle
+        fhss_band_last = fhss_band;
```

Comment:
If they are equal, why set fhss_band_last?

Reply:
ups, that's true. THX.